schema-importer/app/src/main/kotlin/com/scalar/db/analytics/postgresql/schemaimporter/DatabaseContext.kt

```diff
@@ -42,3 +43,10 @@ fun executeUpdateWithLogging(stmt: Statement, logger: KLogger, sql: String) {
     logger.debug { sql }
     stmt.executeUpdate(sql)
 }
+
+fun escapeIdentifier(ident: String): String {
+    return Utils.escapeIdentifier(null, ident).toString()
+}
+fun escapeLiteral(literal: String): String {
```

Comment:
An empty line before this method would be nice.

Reply:
Thanks! I added the blank line in https://github.com/scalar-labs/scalardb-analytics-postgresql/pull/45/commits/9f87de662447482935ff076f39acee5be7ea18ac